oscovida/oscovida.py

```diff
@@ -1590,9 +1585,8 @@ def make_compare_plot_germany(region_subregion,
 
     if dates and weeks == 0:
         try:
-            date_start, date_end = dates.split(':')
-            res_c = df_c[date_start:date_end]
-            res_d = df_d[date_start:date_end]
+            res_c = cut_dates(df_c, dates)
+            res_d = cut_dates(df_d, dates)
             kwargs_c.update({'labels': False})
             kwargs_d.update({'labels': False})
         except ValueError:
```

Comment:
Can this exception still occur (now that we use `cut_dates`)?

Reply:
Totally right! Thanks for noticing! Fixed.